src/container.c

```diff
@@ -113,23 +113,33 @@ static int container_setup_volume(struct hyper_container *container)
 		sprintf(path, "/tmp/%s", vol->mountpoint);
 		sprintf(mountpoint, "./%s", vol->mountpoint);
 
-		fprintf(stdout, "mount %s to %s, tmp path %s\n",
-			dev, vol->mountpoint, path);
-
 		if (hyper_mkdir(path, 0755) < 0) {
 			perror("create volume dir failed");
 			return -1;
 		}
 
-		if (!strncmp(vol->fstype, "xfs", strlen("xfs")))
-			options = "nouuid";
+		if (!strncmp(vol->fstype, "nfs", strlen("nfs"))) {
```

Comment:
could you use strcmp()?
strncmp() is bug here, hyperstart is full of such bugs....